clab/config.go

```diff
@@ -183,6 +183,13 @@ func (c *cLab) kindInitialization(nodeCfg *NodeConfig) string {
 	return c.Config.Topology.Defaults.Kind
 }
 
+func (c *cLab) mountsInit(nodeCfg *NodeConfig) []string {
+	if len(nodeCfg.Mounts) != 0 {
+		return nodeCfg.Mounts
+	}
+	return c.Config.Topology.Kinds[nodeCfg.Kind].Mounts
```

Comment:
This probably needs a check, if key `nodeCfg.Kind` exists in the map

Reply:
since Go will return an empty value for a type that is stored under a missing key in a map, that serves our purpose here

Reply:
ok, it's not a pointer there